core-web/src/main/java/org/silverpeas/core/web/mvc/SilverListener.java

```diff
@@ -111,7 +111,7 @@ public void requestInitialized(final ServletRequestEvent sre) {
     final Runnable setupSessionCache;
     if (sessionInfo.isDefined()) {
       if (sessionInfo instanceof HTTPSessionInfo && !sessionInfo.isAnonymous()) {
-        // a non anonymous HTTP session
+        // a no anonymous HTTP session
```

Comment:
I think it's "non", even if I know that your English is better than mine :-)

Reply:
Yes: non-anonymous